src/pages/core/plugins/index.md

```diff
@@ -62,43 +64,43 @@ If an Permanent Plugin is added to an Asset/Collection without an authority set
 
 ## Collection Plugins
 
-Collection Plugins are plugins that can have a collection wide effect. This is particularly useful for royalties where you should assign the [royalties plugin](/core/plugins/royalties) to the Collection Asset and all nfts in that collection will now reference to that plugin.
+Collection Plugins are plugins that have a collection-wide effect. This is particularly useful for royalties because you can assign the [royalties plugin](/core/plugins/royalties) to the Collection Asset and all Assets in that collection will now reference that plugin.
 
 Collections only have access to `Permanent Plugins` and `Authority Managed Plugins`.
 
 ## Plugin Priority
 
-If a MPL Core Asset and MPL Core Collection Asset both share the same plugin type then the plugin and its data on the Core Asset will take precedence over the Collection Asset plugin.
+If an MPL Core Asset and MPL Core Collection Asset both share the same plugin type then the Asset level plugin and its data will take precedence over the Collection level plugin.
 
-This can be used in creative ways such as setting royalties at different levels for your collection of assets. E.g.:
+This can be used in creative ways like setting royalties at different levels for a collection of assets.
 
 - Collection Asset has a Royalties Plugin assigned at 2%
-- Your Super Rare MPL Core Asset then has a Royalty Plugin assigned at 5%
+- A Super Rare MPL Core Asset within the collection has a Royalty Plugin assigned at 5%
 
-This means all your regular MPL Core Asset sales from your collection will retain a 2% royalty while your Super Rare MPL Core Asset will retain a 5% royalty at sale because it has it's own Royalties Plugin that will take precedence over the Collection Asset Royalties Plugin that is also assigned.
+TIn the above case, regular MPL Core Asset sales from the collection will retain a 2% royalty while the Super Rare MPL Core Asset will retain a 5% royalty at sale because it has it's own Royalties Plugin that takes precedence over the Collection Asset Royalties Plugin.
```

Comment:
```suggestion
In the above case, regular MPL Core Asset sales from the collection will retain a 2% royalty while the Super Rare MPL Core Asset will retain a 5% royalty at sale because it has it's own Royalties Plugin that takes precedence over the Collection Asset Royalties Plugin.
```